requirements.txt

```diff
@@ -1,5 +1,11 @@
+fastapi
```

Comment:
we should kill these dependencies. why are they needed by default? I think for now `run_RAG.sh` can have them? people may be using other examples for which none of these dependencies should be required?

Reply:
we are not using fastapi, this appears to have existed even prior to [this commit](https://github.com/meta-llama/llama-stack-apps/blob/48bf3686e51555b2908ac439880919d93831bc93/requirements.txt)